docs/spec-datasync.md

```diff
@@ -0,0 +1,87 @@
+---
+id: datasync
+title: Data Synchronization
+---
+
+This is an optional part of the spec that is meant to support offline-first GraphQL clients. It outlines the features and specifications needed to have for smooth offline operation.
+
+It consists of two aspects:
+
+- Fetch data that was changed based on the client side timestamp
```

Comment:
DeltaQuery based on the `lastChanged` token

It is not guaranteed to be timestamp